lib/chef/resource/apt_repository.rb

```diff
@@ -33,7 +33,7 @@ class AptRepository < Chef::Resource
       # whether or not to add the repository as a source repo, too
       property :deb_src, [TrueClass, FalseClass], default: false
       property :keyserver, [String, nil, false], default: "keyserver.ubuntu.com", nillable: true, coerce: proc { |x| x ? x : nil }
-      property :key, [String, nil, false], default: nil, nillable: true, coerce: proc { |x| x ? x : nil }
+      property :key, [String, Array, nil, false], default: [], coerce: proc { |x| x ? [x].flatten : nil }
```

Comment:
` x ? [x].flatten : nil`
is better written as
` x ? Array(x) : nil`

Reply:
Thanks! I just referenced what was used elsewhere, powershell, dnf and chocolatey package providers used `[x].flatten`. No problem to change if `Array(x)` is preferred though!